src/main/java/com/codejoust/main/dto/account/AccountUidDto.java

```diff
@@ -0,0 +1,12 @@
+package com.codejoust.main.dto.account;
```

Comment:
The reason for not using AccountDto was to avoid large DTO classes - for example, if a GameDto holds a list of problems, and each problem holds the owner, and each owner contains another list of problems, then that could get exponentially large 

But I think what you said about having the ownerUid as a string (probably mapped manually in the mapper class) makes a lot of sense - I'll implement that!

Reply:
Actually, I just realized one other potential reason for having a separate `AccountDto` and `AccountUidDto` (maybe better named as like `BasicAccountDto`) is once we add demographic and other account info, it might include fields such as name, email, school, etc. while still trimming out unnecessary fields like the list of problems or room histories. What do you think about this?

Reply:
Ok, got it I think that makes sense - we can keep both in 👍